projects/plugins/jetpack/extensions/blocks/ai-assistant/index.js

```diff
@@ -0,0 +1,67 @@
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import { ExternalLink } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+
+/**
+ * Style dependencies
+ */
+import './editor.scss';
+
+export const name = 'ai-assistant';
+export const title = __( 'AI Assistant', 'jetpack' );
+export const settings = {
+	apiVersion: 2,
+	title,
+	description: (
+		<Fragment>
+			<p>{ __( 'Automatically generate and modify content, powered by AI magic.', 'jetpack' ) }</p>
+			<p>
+				{ __(
+					'We are experimenting with this feature and can tweak or remove it at any point.',
+					'jetpack'
+				) }
+			</p>
+			<ExternalLink href={ getRedirectUrl( 'jetpack_ai_feedback' ) }>
+				{ __( 'Share your feedback.', 'jetpack' ) }
+			</ExternalLink>
+		</Fragment>
+	),
+	icon: {
+		src: 'superhero',
+		foreground: getIconColor(),
+	},
+	category: 'text',
+	keywords: [
+		_x( 'AI', 'block search term', 'jetpack' ),
+		_x( 'GPT', 'block search term', 'jetpack' ),
+		_x( 'AL', 'block search term', 'jetpack' ),
+		_x( 'Magic', 'block search term', 'jetpack' ),
+		_x( 'images', 'block search term', 'jetpack' ),
```

Comment:
let's skip these about images and add one for "help"

Reply:
In the same line of "help", maybe "assistant" is useful as well. What do you think?